Integrations/integration-Cisco-umbrella.yml

```diff
@@ -0,0 +1,885 @@
+description: Cisco Umbrella Investigate
+detaileddescription: 'Cisco Investigate is part of the Cisco Umbrella package. When
+  you log into the CIsco Umbrella portal you will need to obtain the API Token for
+  the Cisco Investigate Feature. '
+configuration:
+- display: Please enter your api token
+  name: APIToken
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |2
+
+    var insecure = params.insecure;
+    var proxy = params.proxy;
+    var apitoken = params.APIToken;
+    var baseURL = 'https://investigate.api.umbrella.com';
```

Comment:
Would use as a param with this default value (Is it possible to have umbrella on premise?)